src/app/widget-simple-linear/widget-simple-linear.component.ts

```diff
@@ -0,0 +1,200 @@
+import { ViewChild, Input, ElementRef, Component, OnInit, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { MatDialog } from '@angular/material/dialog';
+
+import { SignalKService } from '../signalk.service';
+import { ModalWidgetComponent } from '../modal-widget/modal-widget.component';
+import { WidgetManagerService, IWidget, IWidgetConfig } from '../widget-manager.service';
+import { UnitsService } from '../units.service';
+import { AppSettingsService } from '../app-settings.service';
+
+const defaultConfig: IWidgetConfig = {
+  displayName: "Display Name",
+  paths: {
+    "gaugePath": {
+      description: "Numeric Data",
+      path: null,
+      source: null,
+      pathType: "number",
+    }
+  },
+  units: {
+    "gaugePath": "V"
```

Comment:
I saw you polished the units convert service to be more UI friendly (Great work btw :D) However I think we'll need to update these defaults too?

Reply:
So basically I reviewed the SignalK specs path by path and made sure our defaults match those, string for string. It fixed a few getPathUnits meta look up and the Widget config Units tab has quite a few more match’s with the meta support you built. I also added a few more units I plan on using - time, electric stuff, etc. and adjusted the Kip Default Units config Tab accordingly.

Second I updated the classes to support the new IUnitGroup format with description. Let me know what you want to change, I’d be happy to fix any issue, or improve on it.

Reply:
Oh, and I also update blank, unitdefault and demo config.